contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/plugins/image-upload/image-html-to-md.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Helper function that converts all image HTML representations
+ * to markdown based on `data-image` attribute contents.
```

Comment:
> based on `data-image` attribute contents

I believe that this part of the comment can be removed - this is the case only for formulas that use `data-formula` attribute to work well (it's related to MathQuill)

Reply:
Ah nice catch!